ozaria/site/components/interactive/PageInteractive/draggableOrdering/index.vue

```diff
@@ -136,6 +153,36 @@
 
       resetAnswer () {
         this.promptSlots = this.getShuffledPrompt(this.shuffle)
+
+        this.initializedAnswer = false
+        this.initializeFromPastSubmission()
+      },
+
+      initializeFromPastSubmission () {
+        if (!this.pastCorrectSubmission || this.initializedAnswer) {
+          return
+        }
+
+        this.initializedAnswer = true
+
+        let missingAnswer = false
```

Comment:
Since we are handling `missingAnswer` within the `map`, should this be initialized for each element?

Reply:
Not sure I follow, can you elaborate?  `missingAnswer` is just used to track if we did find a missing answer instead of searching through the array again for undefined.